src/aap_eda/api/serializers/auth.py

```diff
@@ -72,18 +84,24 @@ class RoleSerializer(serializers.ModelSerializer):
     is_default = serializers.BooleanField(
         default=False, help_text="Indicates if the role is default"
     )
+    created_at = serializers.SerializerMethodField()
+    modified_at = serializers.SerializerMethodField()
+
+    def get_created_at(self, obj):
+        return obj.created
+
+    def get_modified_at(self, obj):
+        return obj.modified
```

Comment:
IMO It's crucial to document the purpose of these methods and its role in satisfying the DAB interface requirements. A detailed docstring would help future developers understand why this method exists and prevent accidental removal if its usage isn't immediately obvious.